src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -19,26 +19,31 @@ public void put(K key, V value) {
         if (size >= threshold) {
             resize();
         }
-        Node<K, V> currentNode = table[getKeyIndex(key)];
+        int index = getIndex(key);
+        Node<K, V> currentNode = table[index];
         if (currentNode == null) {
-            table[getKeyIndex(key)] = new Node<>(key, value, null);
+            table[index] = new Node<>(key, value, null);
```

Comment:
put this line between line 40 and 41 and remove completely this if

Reply:
done